tensorflow/python/training/input.py

```diff
@@ -762,6 +762,9 @@ def _shuffle_batch(tensors, batch_size, capacity, min_after_dequeue,
   tensor_list = _as_tensor_list(tensors)
   with ops.name_scope(name, "shuffle_batch",
                       list(tensor_list) + [keep_input]) as name:
+    if capacity == min_after_dequeue:
+      raise ValueError("capacity %d must not be equal to min_after_dequeue %d."
+                       % (capacity, min_after_dequeue))
```

Comment:
Looking at the implementation in `random_shuffle_queue_op.cc`, it seems there will be a runtime error if `capacity <= min_after_dequeue`. Since you're adding a check here, can you extend it to cover the `<` case as well,  and update the message?